playpen/ansible/roles/dev/files/bashrc

```diff
@@ -72,6 +72,8 @@ _paction() {
 
 psmash() {
     workon pulp-smash;
+    prestart;
+    find ~/devel/pulp-smash/ -name *.pyc -delete;
```

Comment:
`-name *.py[co]` to also account for optimized compilation artifacts.

All these semicolons need to go, but that should be its own PR.